tools/terraform/ecs.tf

```diff
@@ -123,6 +122,37 @@ resource "aws_ecs_task_definition" "backend" {
   memory                   = var.fleet_backend_mem
   container_definitions = jsonencode(
     [
+      {
+        name      = "prometheus-exporter"
+        image     = "917007347864.dkr.ecr.us-east-2.amazonaws.com/prometheus-to-cloudwatch:latest"
```

Comment:
this won't work here. hardcoded account id, we might eventually just publish the image to our own public ecr gallery?